### PR TITLE
API compliance fix and build fixes

### PR DESCRIPTION
**Pull request by kfswain**

### CODE REVIEW

File: pkg/ext-proc/backend/datastore.go [hunk 1]
```diff
@@ -76,7 +76,8 @@ func (s *K8sDatastore) FetchModelData(modelName string) (returnModel *v1alpha1.I
 }
 
 func RandomWeightedDraw(model *v1alpha1.InferenceModel, seed int64) string {
-	weights := 0
+	var weights int32
+	weights = 0
```
Comment (liu-cong) [inline pkg/ext-proc/backend/datastore.go:80]: nit: weights will be defaulted to 0, no need for this.

**Review by liu-cong — COMMENTED** [review]
/lgtm
/approve

**Comment by kfswain** [comment]
Closes #111

**Comment by danehans** [comment]
Fixes #111

**Review by danehans — APPROVED** [review]

**Pull request merged**